Redis.cpp

```diff
@@ -102,9 +102,20 @@ bool Redis::hexists(const char *key, const char *field)
 std::vector<String> Redis::lrange(const char *key, int start, int stop)
 {
   auto rv = RedisCommand("LRANGE", ArgList{key, String(start), String(stop)}).issue(conn);
-  return rv->type() == RedisObject::Type::Array
+
+  if(rv->type() == RedisObject::Type::InternalError)
```

Comment:
Great thought adding handling for this and my bad for not doing so initially! Thank you! 🙇 

However, I worry a bit that this could become an issue for consumers that aren't aware of the implementation and don't explicitly check the first element in the returned vector for an error string, because otherwise it will appear to them as a valid one-element list.

However, I'm realizing now that the interface of this method currently doesn't allow passing out-of-band error information to the consumer _besides_ raising an exception. Fortunately this is, I believe, a valid place to raise an exception as it would be a programming error.

So I think I'd prefer this raise an exception: what do you think @nafraf, @holla2040?